pysper/search/filtercache.py

```diff
@@ -143,7 +144,7 @@ def _get_stats(events, ctor, key_name):
             [
                 event
                 for event in events
-                if event.get("event_type", "") == (key_name + "_duration")
+                if event.get("event_type", "") == "eviction_duration"
```

Comment:
This may not work with the enumerate trick (if events do not have `id`).
It is possible to make it working but I am not sure how important it is...
... maybe instead we should fail fast on lack of ids?

Reply:
well we need to analyze as much of the tarball as possible, I would rather throw away things we can't associate (and log them in warnings)